packages/core/src/contracts/newsroom.ts

```diff
@@ -94,6 +108,10 @@ export class Newsroom extends BaseWrapper<NewsroomContract> {
     return this.multisigProxy.owners();
   }
 
+  public async owner(): Promise<EthAddress> {
```

Comment:
Bad idea. We might have some different way of ACL on Newsroom in the future. Using just owners abstracts it away.

If you wanna check for multisig, add `hasBoardOfDirectors(): Promise<EthAddress?>` or something like this and add this functionallity to `BaseMultisigProxy` (`return isDefined(this.multisig);`);